src/v2/guide/migration.md

```diff
@@ -1462,22 +1462,22 @@ Use the [Vue Devtools](https://github.com/vuejs/vue-devtools) for the optimal de
 </div>
 {% endraw %}
 
-## Instance DOM Options
+## Opciones de la instancia DOM
 
-### `replace: false` <sup>removed</sup>
+### `replace: false` <sup>removido</sup>
 
-Components now always replace the element they're bound to. To simulate the behavior of `replace: false`, you can wrap your root component with an element similar to the one you're replacing. For example:
+Los componentes ahora siempre reemplazan al elemento al que están vinculados. Para simular el comportamiento de `replace: false`, usted puede ajustar su componente raíz con un elemento similar al que está reemplazando. Por ejemplo:
 
-``` js
+```js
 new Vue({
   el: '#app',
   template: '<div id="app"> ... </div>'
 })
 ```
 
-Or with a render function:
+O una función render:
```

Comment:
What do you think about changing this to: `O una función de renderizado:`

Reply:
I thought about that, but then thought that the function is called `render` so maybe was better to keep it in the same way (?)

I'm not sure about what is the best option here

Reply:
Maybe we can wrap "render" between backticks 🤔 (``O una función de `render`:``)

Reply:
for me `O _con_ una función de renderizado:` is okay